lib/pltr/v2/helpers/cards.js

```diff
@@ -24,8 +24,13 @@ export function cardMapping(sortedBeats, sortedLines, card2Dmap, currentLine) {
 }
 
 function lineIsHidden(line, currentLine) {
-  if (!currentLine) return false
-  return line.id != currentLine
+  // if no outlinefilter return all
+  if (Array.isArray(currentLine) && !currentLine.length) return !currentLine
+  if (Array.isArray(currentLine) && currentLine.length) return !currentLine.includes(line.id)
+  // if file has no outlineFilter saved in redux
+  else if (!currentLine) {
+    return false
+  } else if (!!currentLine) return line.id != currentLine
```

Comment:
I think that this line should instead read:
```
else return line.id !== currentLine
```
because we know at this point that `currentLine` must be truthy.